cmd/cluster-kube-scheduler-operator/render/render.go

```diff
@@ -73,8 +73,7 @@ func NewRenderCommand() *cobra.Command {
 	cmd.Flags().StringVar(&renderOpts.assetOutputDir, "asset-output-dir", "", "Output path for rendered manifests.")
 	cmd.Flags().StringVar(&renderOpts.assetInputDir, "asset-input-dir", "", "A path to directory with certificates and secrets.")
 	cmd.Flags().StringVar(&renderOpts.templatesDir, "templates-input-dir", "/usr/share/bootkube/manifests", "A path to a directory with manifest templates.")
-	cmd.Flags().StringVar(&renderOpts.configOverrideFile, "config-override-file", "", "A sparse KubeSchedulerConfiguration."+
-		"componentconfig/v1alpha1 file (default: kube-scheduler-config-overrides.yaml in the asset-input-dir)")
+	cmd.Flags().StringVar(&renderOpts.configOverrideFile, "config-override-file", "", "An additional sparse KubeSchedulerConfig.kubecontrolplane.config.openshift.io/v1 file for customiziation through the installer.")
```

Comment:
this KubeSchedulerConfig.kubecontrolplane.config.openshift.io/v1 looks incorrect?

Reply:
What is the correct one?

Reply:
for all kube-scheduler configs, shouldn't it be " componentconfig/v1alpha1"? or am I missing something?